installscripts/jazz-terraform-unix-noinstances/scripts/gitlab-trigger-job.sh

```diff
@@ -0,0 +1,159 @@
+JENKINS_URL=http://$1/ # localhost or jenkins elb url
+JOB_NAME="Gitlab-Trigger-Job"
+USERNAME=
+PASSWORD=
+
+JENKINS_CLI=./jenkins-cli.jar
+
+cat <<EOF | java -jar $JENKINS_CLI -s $JENKINS_URL -auth $USERNAME:$PASSWORD create-job $JOB_NAME
+<?xml version='1.0' encoding='UTF-8'?>
+<flow-definition plugin="workflow-job@2.12">
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <com.dabsquared.gitlabjenkins.connection.GitLabConnectionProperty plugin="gitlab-plugin@1.5.2">
+      <gitLabConnection>Jazz-Gitlab</gitLabConnection>
+    </com.dabsquared.gitlabjenkins.connection.GitLabConnectionProperty>
+    <org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+      <triggers>
+        <com.dabsquared.gitlabjenkins.GitLabPushTrigger plugin="gitlab-plugin@1.5.2">
+          <spec></spec>
+          <triggerOnPush>true</triggerOnPush>
+          <triggerOnMergeRequest>true</triggerOnMergeRequest>
+          <triggerOnPipelineEvent>false</triggerOnPipelineEvent>
+          <triggerOnAcceptedMergeRequest>false</triggerOnAcceptedMergeRequest>
+          <triggerOnClosedMergeRequest>false</triggerOnClosedMergeRequest>
+          <triggerOpenMergeRequestOnPush>never</triggerOpenMergeRequestOnPush>
+          <triggerOnNoteRequest>true</triggerOnNoteRequest>
+          <noteRegex>Jenkins please retry a build</noteRegex>
+          <ciSkip>true</ciSkip>
+          <skipWorkInProgressMergeRequest>true</skipWorkInProgressMergeRequest>
+          <setBuildDescription>true</setBuildDescription>
+          <branchFilterType>All</branchFilterType>
+          <includeBranchesSpec></includeBranchesSpec>
+          <excludeBranchesSpec></excludeBranchesSpec>
+          <targetBranchRegex></targetBranchRegex>
+          <secretToken>replace</secretToken>
+        </com.dabsquared.gitlabjenkins.GitLabPushTrigger>
+      </triggers>
+    </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
+  </properties>
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@2.36">
+   <script>#!groovy.
+
+node  {
```

Comment:
Why is this code here? Pai had multiple bugs to this code which he submitted on the core side and part of the PR he addressed them: https://github.com/tmobile/jazz/commit/0760d2ba1ce2da149f3dc81d983c5e3617a4bb13#diff-8ea6452f226bf245567e69f9c89ac9d9L28. 

I see the buggy code here. Is this the right approach where we have the groovy script embedded as part of installer when a Jenkins file exists on core? @Feroz-Shaikh : Let me know how what you think